deployment/scripts/run_dcl_node

```diff
@@ -0,0 +1,459 @@
+#!/bin/sh
+# Copyright 2020 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+set -o pipefail
+
+export PATH="$PATH:/usr/bin"
+
+SCRIPT_PATH="$(readlink -f "$0")"
+BASEDIR="$(dirname "$SCRIPT_PATH")"
+
+# TODO DOC dcld.service and other files in the current directory
+# TODO errors to stderr
+# TODO optionally fetch dcld, dclcli and dcld.service from GitHub release
+
+SERVICE_FILE="./dcld.service"
+PERSISTENT_PEERS_FILE='./persistent_peers.txt'
+GENESIS_IN_FILE="./genesis.json"
+CONFIG_FILE="$HOME/.dcld/config/config.toml"
+GENESIS_FILE="$HOME/.dcld/config/genesis.json"
+
+CLI_PORT=26657
+
+PERSISTENT_PEERS=
+PEER=
+PEER_ID=
+PEER_HOSTSPEC="localhost:$CLI_PORT"
+
+
+# ================ FUNCTIONS ============================
+DEF_NODE_TYPE="validator"
+DEF_CHAIN_ID="testnet"
+DEF_NODE_USER="ubuntu"
+
+# NODE_TYPE="${1:-validator}"
+# NODE_NAME="$2"
+# KEY_NAME="${3:-${NODE_NAME}admin}"
+# CHAIN_ID="${4:-testnet}"
+# NODE_USER="${5:-ubuntu}"
+# PEER="${5:-}"
+
+verbosity=0
+NODE_NAME=
+KEY_NAME=
+PEER=
+
+NODE_TYPE="$DEF_NODE_TYPE"
+CHAIN_ID="$DEF_CHAIN_ID"
+NODE_USER="$DEF_NODE_USER"
+
+
+function usage {
+  echo "\
+Usage: $0 [options] node-name
+
+Initializes DCL node of a given type.
+
+Options:
+  -c,  --chain-id   chain id, default: '$DEF_CHAIN_ID'
+  -h,  --help       print this help and exit
+  -k,  --key-name   account key-name, default: '<node-name>admin'
+  -p,  --peer       peer hostspec as 'HOST:PORT', default: first entry of 'persistent_peers.txt'
```

Comment:
Not sure that this is a correct default. I believe the local CLI should be always connected to localhost, shouldn't it?
If it's not a localhost, then it should be the node owned by that user/company, and not the first entry from `persistent_peers.txt`.

Reply:
hid that for now